fetch.bs

```diff
@@ -1404,10 +1407,16 @@ Unless stated otherwise, it is unset.
 <div class="note no-backref">
  <dl>
   <dt>"<code>default</code>"
-  <dd><a for=/>Fetch</a> will inspect the HTTP cache on the way to the network.
-  If there is a fresh response it will be used. If there is a stale response a conditional request
-  will be created, and a normal request otherwise. It then updates the HTTP cache with the response.
-  [[!HTTP]] [[!HTTP-SEMANTICS]] [[!HTTP-COND]] [[!HTTP-CACHING]] [[!HTTP-AUTH]]
+  <dd><a for=/>Fetch</a> will inspect the HTTP cache on the way to the network:
+  <ol>
+   <li><p>If the HTTP cache contains a matching <a>fresh response</a> it will be returned.
+   <li><p>If the HTTP cache contains a matching <a>stale-while-revalidate response</a> it will be returned,
+   and a conditional network fetch will be made to update the entry in the HTTP cache.
+   <li><p>If the HTTP cache contains a matching <a>stale response</a>, a conditional network will be created
+   and returned.
```

Comment:
This drops the cache updating that the original text had. (Same for the next point.)

Also, perhaps this should be a long paragraph instead similar to the original? It's not really meant to be an algorithm. Just an explanation.

Reply:
done